package.json

```diff
@@ -38,6 +38,7 @@
     "seed": "node scripts/seed-local-env.js",
     "setup": "pnpm --filter @hive/storage setup",
     "test": "jest",
+    "test:e2e": "CYPRESS_BASE_URL=$HIVE_APP_BASE_URL cypress run",
```

Comment:
how locally this should work? should I add `HIVE_APP_BASE_URL` in root's `.env`? maybe we need to add `.env.template` with this variable?

Reply:
We need it in the root `.env.template`, I've pretty much followed the required envs in https://docs.graphql-hive.com/self-hosting/get-started.